doc/installation/mac_install.rst

```diff
@@ -49,7 +49,7 @@ Manual installation
 
 The clang/clang++ compiler that ships with OS X/macOS does not support OpenMP threads and creates code that fails some tests. You therefore need to use **GCC** to compile NEST under OS X/macOS.
 
-Installation instructions here have been tested under OS X 10.11 *El Capitan* and macOS 10.12 *Sierra* with `Anaconda Python 2 and 3 <https://www.continuum.io/anaconda-overview>`_ and all other dependencies installed via `Homebrew <http://brew.sh>`_. See below for `Manual installation with dependencies from MacPorts`_.
+Installation instructions here have been tested under macOS 10.14 *Mojave* with `Anaconda Python 3 <https://www.continuum.io/anaconda-overview>`_ and all other dependencies installed via `Homebrew <http://brew.sh>`_. They should also work with earlier versions of macOS.
```

Comment:
@heplesser Can you be more explicit here in the steps you took to create this environment? 
What about the PyNEST dependencies like numpy, scipy, and matplotlib?
which order did you use (brew install then conda install) ? Do you create a conda environment before installing the dependencies?

Reply:
I have tried different approaches at different times, both with the full Anaconda package and miniconda plus the `conda install` line shown in the Python section. I did not explicitly create conda environments in all cases I tested, it worked with and without for me. Therefore I have't been more specific here.